scootapi/client/run_job_cmd.go

```diff
@@ -42,17 +45,18 @@ type CLIJobDef struct {
 	JobType              string
 	Requestor            string
 }
+
 type TaskDef struct {
 	Args       []string
 	SnapshotID string
-	JobID      string
-	TaskID     string
 	TimeoutMs  int32
+	tags.LogTags
 }
 
 func (c *runJobCmd) run(cl *simpleCLIClient, cmd *cobra.Command, args []string) error {
 	log.Info("Running on scoot, args:", args)
 	jobDef := scoot.NewJobDefinition()
+	jobDef.RequestorTag = &c.RequestorTag
```

Comment:
What's happening with the rest of the log tags here?

Reply:
To add to that - we don't have the jobID until cl.scootClient.RunJob(jobDef) returns

Reply:
In that case, should we be using LogTags at all in TaskDef? What we do on the client side doesn't have to match the structs used by scheduler and worker which have additional context and can supplement the request with jobId etc.

Reply:
Fair enough - guess not!